cmd/minikube/cmd/console.go

```diff
@@ -47,14 +51,26 @@ var consoleCmd = &cobra.Command{
 		}
 		if consoleURLMode {
 			fmt.Fprintln(os.Stdout, url)
+		} else if machineReadAble{
+			hostIP, err := cluster.GetHostIP(api)
```

Comment:
Design wise, I find it odd that in this method you read the url above the if/switch and the ip in this sub-branch. But never mind